src/stream-chat-net/Client.cs

```diff
@@ -63,16 +63,16 @@ private Client(ClientOptions opts, HttpClient httpClient, string apiKey, string
             };
             _token = this.GenerateJwt(payload);
             var assemblyVersion =  typeof(Client).GetTypeInfo().Assembly.GetName().Version;
+
             Version = string.Join(".", assemblyVersion.Major, assemblyVersion.Minor, assemblyVersion.Build);
+            Users = new Users(this);
+            Permissions = new PermissionService(this);
         }
 
-        public IUsers Users
-        {
-            get
-            {
-                return new Users(this);
```

Comment:
Why create a new class every time. We can just create one in the constructor.

Reply:
Exactly, this is good. It's probably overseen as http client recreation